test/functional/api/cas/casadm.py

```diff
@@ -42,6 +42,58 @@ def start_cache(cache_dev: Device, cache_mode: CacheMode = None,
     return Cache(cache_dev)
 
 
+def standby_init(cache_dev: Device, cache_id: int, cache_line_size: CacheLineSize,
+                 force: bool = False, shortcut: bool = False,
+                 kernel_params: KernelParameters = KernelParameters()):
+    if kernel_params != KernelParameters.read_current_settings():
+        reload_kernel_module("cas_cache", kernel_params.get_parameter_dictionary())
+    output = TestRun.executor.run(
+        standby_init_cmd(
+            cache_dev=cache_dev.path,
+            cache_id=str(cache_id),
+            cache_line_size=str(cache_line_size),
+            force=force,
+            shortcut=shortcut,
+        )
+    )
+    if output.exit_code != 0:
+        raise CmdException("Failed to init standby cache.", output)
+    return Cache(cache_dev)
+
+
+def standby_load(cache_dev: Device, shortcut: bool = False,
+                 kernel_params: KernelParameters = KernelParameters()):
+    if kernel_params != KernelParameters.read_current_settings():
+        reload_kernel_module("cas_cache", kernel_params.get_parameter_dictionary())
```

Comment:
drop reloading kernel modules when loading cache (we don't want to change the settings when we're restoring the cache)